unit-tests/run-test.py

```diff
@@ -0,0 +1,69 @@
+#!python3
+
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+import sys, os, subprocess, re, platform
+
+
+def usage():
+    ourname = os.path.basename(sys.argv[0])
+    print( 'Syntax: ' + ourname + ' <test-to-run>' )
+    sys.exit(2)
+
+# Parse command-line:
+test_name = None
+try:
+    test_name = sys.argv[1]
+except IndexError as err:
+    print( err )
+    usage()
+
+def find_file( root, mask ):
```

Comment:
Needs description
As I read it, it just returns the *first* match, not all of them, yes?

Reply:
correct, I do not expect to find more than 1

Reply:
Would be nice if it returns all of them (`yield`), then you can do `run-test.py test-algo-*.py`
Also, you have to include the `.py` part, right?

Reply:
Finally, can't we just combine this into run-unit-tests.py, so there aren't two different scripts?
I.e., if you run `run-unit-tests.py test-1.py` then it would see that `test-1.py` is not a directory and will try to find an actual test (or tests)

Reply:
Good idea, i'll change to yield. If you don't add the .py it will try to run other files as well.